README.md

```diff
@@ -57,6 +57,14 @@ After installation and bundler configuration all of the components are ready to
 
 Just include `import { Button } from '@silintl/ui-components'` in your script section to import Button and any other components you want to use.
 
+## overriding default sass variables
+
+In your main sass file just use the following pattern to override and sass variables. See <https://github.com/silinternational/ui-components/blob/develop/stories/_index.scss> for exposed sass variables.
```

Comment:
Did you intend to use angle brackets rather than normal markdown square brackets?

Reply:
I'm not sure what I meant, but I don't need these angle brackets.